types/string-algorithms/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for string-algorithms 1.0
+// Project: https://github.com/burgaard/string-algorithms
+// Definitions by: lionel-rowe <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.0
```

Comment:
```suggestion
```
not enforced:
https://github.com/DefinitelyTyped/DefinitelyTyped#support-window

Reply:
As with the similar comment on https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63686/, removing this causes errors when I run `npm test string-algorithms` locally:

```log
Error: /root/source/repos/DefinitelyTyped/types/string-algorithms/index.d.ts:10:12
ERROR: 10:12  expect  Compile error in typescript@3.9 but not in typescript@4.0.
Fix with a comment '// Minimum TypeScript Version: 4.0' just under the header.
Cannot find name 'entries'.
[+ 7 similar errors]
/root/source/repos/DefinitelyTyped/types/string-algorithms/string-algorithms-tests.ts:7:1
ERROR: 7:1    expect  Compile error in typescript@3.9 but not in typescript@4.0.
Fix with a comment '// Minimum TypeScript Version: 4.0' just under the header.
Expected 4 arguments, but got 1.
[+ 4 similar errors]
```

In this case, they're all related to `radixSort`, which takes a variable number of parameters depending on whether the 1st parameter is already an array of `number[]` or needs to be converted first.

Reply:
I see, that's named tuple. But people are already using here named tuples without TS header. Again, the same passes locally. I'd make sure latest changes from DT are in your version and local deps are reinstalled.
if issue persists, let's do not crush copies on this. Just close this comment

Reply:
@peterblazejewicz re-pulling from `master` didn't fix it, but `rm -r node_modules ; npm i` did.